text/0000-ownership-transfer.md

```diff
@@ -0,0 +1,118 @@
+- Feature Name: ownership-transfer
+- Start Date: 2020-3-19
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+gem owner -a/r command is used to add or remove owners by an email. We will add options for adding/removing gem owners via Web UI. Additionally, we will make owner add/remove events notifiable, i.e. users can opt-in to receive email notification about ownership changes of a gem. 
+
+We will also add a new flow for ownership transfers. Any gem owner `looking for maintainers` will be able to create `ownership request`. Any user can make an `ownership application` for any gem with less than 100,000 downloads and older than 12 months updated at time or gems which are marked as looking for maintainers. When a gem owner approves an `ownership application`, the requester would be added as an owner to the gem. These `ownership requests` will be searchable and listed on a site wide link.
+
+# Motivation
+
+Option of adding and removing owners through the web UI would facilitate friction-less transfer of gem ownership without dependency on CLI configuration on the user-side.
+
+In the current ownership flow, a gem owner can add a new owner without any confirmation from the user being added as an owner. To avoid this spamming, we will add a requirement of confirmation from the user being added.
+
+New ownership transfer flow will make the process more transparent for audits. Gem owners who are looking for new maintainers/owners would be able to open `ownership requests` and let other interested users fill it to show their interest. New searchable view of ownership transfers will make it easier for the users to find the gems which need maintainers.
+
+# Guide-level explanation
+
+### Change in the ownership flow
+When a new owner is added to a gem, two types of notifications will be sent via email.
+1. A notification to all the existing gem owners about addition of new owner.
```

Comment:
notification to all owners will be send after user has confirmed owner addition.